test_pad_frame/test_pad_frame_genesis2.py

```diff
@@ -0,0 +1,29 @@
+from pad_frame import pad_frame_genesis2
+import glob
+import os
+
+
+def teardown_function():
+    for item in glob.glob('genesis_*'):
+        os.system(f"rm -r {item}")
+
+
+def test_main(capsys):
+    argv = [
+        "--config_data_width", "32",
+        "--config_addr_width", "32",
+        "--num_ios_per_group", "16",
+        "--num_groups_per_side", "1",
+        "--tile_id_offset", "400",
+        "pad_frame/genesis/pad_frame.vp",
+        "pad_frame/genesis/io_group.vp",
+        "pad_frame/genesis/io1bit.vp",
+        "pad_frame/genesis/fixed_io.vp"
+    ]
+    pad_frame_genesis2.pad_frame_wrapper.main(
```

Comment:
nit: can be one line

Reply:
Are you just saying that I should move the close bracket up to the previous line?

Reply:
Nvm you're talking about the last line in that snippet.